log/log.go

```diff
@@ -124,19 +125,27 @@ func (w *bothWriter) logAtLevel(level syslog.Priority, msg string) {
 	const red = "\033[31m\033[1m"
 	const yellow = "\033[33m"
 
-	switch level {
+	switch syslogAllowed := int(level) <= w.syslogLevel; level {
 	case syslog.LOG_ERR:
-		err = w.Err(msg)
-		prefix = red + "E"
+		if syslogAllowed {
```

Comment:
If would move the prefix assignments outside of the `if` clause, since they don't relate to whether syslog is allowed at a given level.